sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1030,6 +1037,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val ENABLE_VECTORIZED_HASH_MAP =
```

Comment:
We need to expose these two parameters? At least, I think it is worthless to add this one...

Reply:
Since this is `.internal()`, this will not be exposed.

Reply:
ok, thanks for the checks.